keyboards/tanuki/tanuki.h

```diff
@@ -0,0 +1,22 @@
+#include "quantum.h"
+
+
+// readability
+#define XXX KC_NO
+
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define KEYMAP( \
```

Comment:
Please change this from `KEYMAP` to `LAYOUT` and don't forget to propagate it to your keymap.c files.